cmd/secretd/root.go

```diff
@@ -330,76 +306,3 @@ func exportAppStateAndTMValidators(
 
 	return wasmApp.ExportAppStateAndValidators(forZeroHeight, jailWhiteList)
 }
-
-// writeParamsAndConfigCmd patches the write-params cmd to additionally update the app pruning config.
-func updateTmParamsAndInit(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
-	cmd := genutilcli.InitCmd(mbm, defaultNodeHome)
-	originalFunc := cmd.RunE
-
-	wrappedFunc := func(cmd *cobra.Command, args []string) error {
-		ctx := server.GetServerContextFromCmd(cmd)
-
-		// time is in NS
-		ctx.Config.Consensus.TimeoutPrecommit = 2_000_000_000
-
-		appConfigFilePath := filepath.Join(defaultNodeHome, "config/app.toml")
-		appConf, _ := serverconfig.ParseConfig(viper.GetViper())
-		appConf.MinGasPrices = "0.25uscrt"
-
-		serverconfig.WriteConfigFile(appConfigFilePath, appConf)
-
-		if err := originalFunc(cmd, args); err != nil {
-			return err
-		}
-
-		return nil
-	}
-
-	cmd.RunE = wrappedFunc
-	return cmd
-}
-
-func initConfig(ctx *client.Context, cmd *cobra.Command) error {
-	cmd.PersistentFlags().Bool(flagLegacyHdPath, false, "Flag to specify the command uses old HD path - use this for ledger compatibility")
```

Comment:
Where all of this is going to?

Reply:
better question: what is it for?

It was showing as unused, that's why it was taken out.